components/telemetry-operator/config/samples/telemetry_v1alpha1_logpipeline_http.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: v1
```

Comment:
I think splitting the file would be nice here as one can directly apply only http logpipeline by few adjustments